spec.idl

```diff
@@ -331,6 +331,11 @@ interface ShorthandProperty : ObjectProperty {
   attribute IdentifierExpression name;
 };
 
+// `PropertyDefinition :: ... AssignmentExpression`
+interface SpreadProperty : ObjectProperty {
+  attribute AssignmentExpression expression;
```

Comment:
`attribute AssignmentExpression expression;`

Reply:
The spec says it's an `AssignmentExpression`. Is the meaning different between our AST and the spec for `AssignmentExpression`?

Reply:
Our spec omits intermediate parse nodes, which means that there can be an Expression in that position (by putting parentheses around a sequence).

Reply:
Fixed